packages/service/src/glint-service.ts

```diff
@@ -71,10 +70,12 @@ export class GlintService implements Service {
    */
   getSourceFile(fileName: string): ts.SourceFile {
     if (isGlintFile(this, fileName)) {
-      return createSyntheticSourceFile(this.ts, {
-        filename: fileName,
-        contents: this.getFileText(fileName),
-      });
+      return ts.createSourceFile(
+        fileName,
+        this.getFileText(fileName),
+        ts.ScriptTarget.Latest,
+        true
```

Comment:
This `true` makes SourceFile object similar to those we got from LanguageService with getStart(), getText() etc. and now they are compatible with TS utility functions and almost compatible with our own codefixes

Reply:
Unsure why we had the synthetic file in the past?

Reply:
We still have synthetic file, but don't need to populate `text: source.contents` because it's done because the `true` in the 4th param.